### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by CarlQGan**

### CODE REVIEW

File: aimsun_scripts/README.md [hunk 1]
```diff
@@ -22,10 +22,10 @@ To create and execute the microsimulation in Aimsun, you must have a Aimsun base
 
 Once the Aimsun base network file and input data is available, please follow the following steps:
 
-1. Change `__REPO_PATH` in `append_github_path_to_python.py` to your absolute path for this repository (traffic-microsimulation).
+1. Change `__REPO_PATH` in `append_github_path_to_python.py` to your absolute path for this repository (traffic-microsimulation). Escape any special characters if necessary.
 2. Change `MICRO_BASELINE = True` in `create_simulations.py` and `run_simulation.py`.
 3. Open Aimsun base network file in Aimsun.
-4. Import all Python files in Aimsun by clicking File > Import > Python Script.
+4. Import all Python files in `aimsun_scripts`, `calibration`, and `utils` into Aimsun by clicking File > Import > Python Script.
```
Comment (johnlee-jh) [inline aimsun_scripts/README.md]: The user only needs to import files in the aimsun_scripts folder within Aimsun.
Could you modify this line to only include aimsun_scripts?

File: aimsun_scripts/README.md [hunk 2]
```diff
@@ -22,10 +22,10 @@ To create and execute the microsimulation in Aimsun, you must have a Aimsun base
 
 Once the Aimsun base network file and input data is available, please follow the following steps:
 
-1. Change `__REPO_PATH` in `append_github_path_to_python.py` to your absolute path for this repository (traffic-microsimulation).
+1. Change `__REPO_PATH` in `append_github_path_to_python.py` to your absolute path for this repository (traffic-microsimulation). Escape any special characters if necessary.
```
Comment (johnlee-jh) [inline aimsun_scripts/README.md]: This is specific to the OS that the user is using. I believe it is a fair assumption that the user can create correct paths for their OS on their own, and thus the addition is not necessary.
Please let me know if you disagree!

**Review by johnlee-jh — COMMENTED** [review]
Hi Carl, thank you for testing the repo and updating the readme. I left two comments, could you please take a look at them? Thanks!

File: aimsun_scripts/README.md (hunk 1 above)
Comment (CarlQGan) [reply aimsun_scripts/README.md]: Got it.

File: aimsun_scripts/README.md (hunk 2 above)
Comment (CarlQGan) [reply aimsun_scripts/README.md]: I agree, it's a more detailed spec for the audience

**Pull request merged**